contrib/kube-prometheus/jsonnet/kube-prometheus/kube-prometheus-static-etcd.libsonnet

```diff
@@ -0,0 +1,95 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+
+(import 'etcd-mixin/mixin.libsonnet') + {
```

Comment:
Why do we need the `etcd-mixin` here? It only gives us alerting and dashboarding definitions, right?

Reply:
Exactly, it gives us dashboards and alerting rules :slightly_smiling_face: .